packages/sdk/test/evm/before-setup.ts

```diff
@@ -215,7 +235,8 @@ export const mochaHooks = {
           factories.push(Marketplace__factory);
           break;
         case PackInitializer.contractType:
-          factories.push(Pack__factory);
+          // disabled temporarily -- size issue in dynamic contracts pattern exetnsions
+          // factories.push(Pack__factory);
```

Comment:
@nkrishang @kumaryash90 we can't have this - we need to port our current extensions to dynamic storage one by one instead of all at once (which means duplicating code for a bit, until we've ported everything)